### PR TITLE
Add guide for soju IRC bouncer.

### LINKED ISSUE #1218: [soju] new guide
I want to write a guide for the [soju IRC bouncer](https://soju.im).
It's an actively maintained IRC bouncer written in go. Setup is pretty easy, so I will publish the PR with the guide soon.

### PR DESCRIPTION
**Pull request by apoxa**
Closes #1218.

### CODE REVIEW

File: source/guide_soju.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+Installation
+============
+
+Clone the source into your home directory and enter the directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://git.sr.ht/~emersion/soju
+ [isabell@stardust ~]$ cd ~/soju
```
Comment (noave) [inline source/guide_soju.rst:49]: the cd command should be moved to the next code-block

File: source/guide_soju.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+Configure soju
+--------------
+
+Save the following as your soju configuration in ``~/etc/soju/config``.
+
+.. code-block:: ini
+
+ listen ircs://0.0.0.0:40132
+ tls /home/isabell/etc/certificates/isabell.uber.space.crt /home/isabell/etc/certificates/isabell.uber.space.key
```
Comment (noave) [inline source/guide_soju.rst:73]: is it possible to use the tilde ~ expansion for /home/isabell here?

File: source/guide_soju.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+You can add additional users with the same command or later via the BouncerServ_ builtin service.
+
+Connect your client
+-------------------
+
+Add a new server in your local IRC client:
+
+* Hostname: `<username>.uber.space`
+* Port: your port
+* Username: soju nickname
```
Comment (noave) [inline source/guide_soju.rst:115]: it should be better referenced, that these nickname/passwords are ones set before

File: source/guide_soju.rst (hunk 2 above)
Comment (apoxa) [reply source/guide_soju.rst:73]: I tested this and unfortunately it's neither possible to use ~ nor $HOME for the path here.

File: source/guide_soju.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+=======
+
+All relevant legal information can be found here:
+
+  * https://git.sr.ht/~emersion/soju/tree/master/item/LICENSE
+
+Prerequisites
+=============
+
+.. include:: includes/open-port.rst
```
[the remark this answers is not in the document]
Comment (apoxa) [reply source/guide_soju.rst:39]: I added a reminder for that and referenced it later.

File: source/guide_soju.rst (hunk 1 above)
Comment (apoxa) [reply source/guide_soju.rst:49]: Yea, that make sense in the overall layout :)

File: source/guide_soju.rst [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+If it's not in state RUNNING, check your configuration.
+
+Create a user
+-------------
+.. note:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ ./.local/bin/sojuctl -config $HOME/etc/soju/config create-user isabell -admin
```
[the remark this answers is not in the document]
Comment (apoxa) [reply source/guide_soju.rst:100]: I replaced that in the command and put a description underneath.

File: source/guide_soju.rst [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+You can also append an IRC server to your username. This is called the Single Upstream Mode and will automatically connect you to this network.
+If you don't do this, you will use the Multiple Upstream Mode.
+Check the manpage_ for more information about this.
+
+Updates
+=======
+
+.. note:: Check the gitrefs_ regularly to stay informed about the newest version.
+
+To update an existing installation remove the ``~/soju`` directory (``rm -rf ~/soju``) and repeat the **Installation** steps. Execute ``supervisorctl restart soju`` to start the new version.
```
[the remark this answers is not in the document]
Comment (apoxa) [reply source/guide_soju.rst:128]: Yeah, my solution is not as clean. I described the process better.

File: source/guide_soju.rst [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://git.sr.ht/~emersion/soju
+
+Compile and install soju to your home directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd ~/soju
+ [isabell@stardust soju]$
```
Comment (noave) [inline source/guide_soju.rst:57]: no need for empty prompt here

**Review by noave — COMMENTED** [review]
Still some things which could be addressed, but no merge-blockers

**Comment by noave** [comment]
Unfortunately I could not successfully run the build:

```
[noaguide@perseus soju]$ PREFIX="/.local" DESTDIR="$HOME" make install
mkdir -p /home/noaguide/.local/bin
mkdir -p /home/noaguide/.local/share/man/man1
mkdir -p /home/noaguide/etc/soju
mkdir -p /home/noaguide/var/lib/soju
cp -f soju sojuctl /home/noaguide/.local/bin
cp -f doc/soju.1 /home/noaguide/.local/share/man/man1
cp: cannot stat ‘doc/soju.1’: No such file or directory
make: *** [Makefile:26: install] Error 1
```

**Comment by luto** [comment]
Thanks for your guide! We would like to keep the list of open pull requests clean to improve the overview. Since this has been dormant for a few months, I'm closing this now. Once you find time to look at the failing build, feel free to submit it again.

**Pull request closed without merging**